extensions/sequelize/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@loopback/sequelize",
+  "version": "0.0.1",
+  "description": "@loopback/sequelize",
+  "keywords": [
+    "loopback-extension",
+    "loopback",
+    "sequelize-connector",
+    "sequelize"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": "14 || 16 || 17 || 18"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run rebuild",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache",
+    "rebuild": "npm run clean && npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/loopbackio/loopback-next.git",
+    "directory": "extensions/sequelize"
+  },
+  "author": "LoopBack contributors",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "peerDependencies": {
+    "@loopback/core": "^4.0.4",
+    "@loopback/repository": "^5.0.4"
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
```

Comment:
debug should be part of devdeps

Reply:
debug appears to be in `dependencies` of all the core loopback packages and other extensions too.